Azure/azure-cli
# [APIM] az apim restore: New command to restore a backup of an API Management service

**Description**
The current Azure CLI has a preview for APIM commands. One of the commands is to back-up a APIM instance, but the restore command is not implemented. This PR is for adding the restore command to Azure CLI. This will fix issue #14114

It is implemented in the same way as the back-up command is implemented.

Remark: Setting up an dev environment on Windows is not easy with your steps. I ended up with using WSL and VSCode remoting. 



**Testing Guide**
Unfortunately I have absolutely no how tests are being done. I see that there some test files (scenario's) with responses from the API, but I have no clue how to get data in the file. 

Also trying to run the tests (`azdev test apim`) in console resulted in a whole bunch of errors.

Please guide me how to add tests and I will try to add them.


**History Notes**


---

This checklist is used to make sure that common guidelines for a pull request are followed.
